src/App.js

```diff
@@ -5,10 +5,12 @@ import "./App.css"
 import { Routes, Route } from 'react-router-dom';
 import WizkidDetails from "./pages/WizkidDetails";
 import WizkidEdit from "./pages/WizkidEdit";
+import Login from "./pages/Login";
+import { useState } from "react";
 
 function App() {
 
-  const {fetchWizkids} = useContext(WizkidsContext);
+  const {fetchWizkids, token} = useContext(WizkidsContext);
```

Comment:
Might be interesting/useful to create a separate context for authentication stuff, like `token`.